pos_picking_delayed/tests/test_module.py

```diff
@@ -12,10 +12,25 @@ def setUp(self):
         self.PosOrder = self.env['pos.order']
         self.QueueJob = self.env['queue.job']
         self.pos_product = self.env.ref('point_of_sale.whiteboard_pen')
-        self.pricelist = self.env.ref('product.list0')
+        self.pricelist = self.env['product.pricelist'].create({
+            'name': 'Test pricelist',
+            'item_ids': [(0, 0, {
+                'applied_on': '3_global',
+                'compute_price': 'formula',
+                'base': 'list_price',
+            })]
+        })
+        self.env['res.partner'].create({
```

Comment:
Why creating this partner if you are not going to use it?

Reply:
I thought that was important, I removed yet